ui/pages/swaps/select-quote-popover/select-quote-popover.stories.js

```diff
@@ -1,6 +1,5 @@
 import React, { useState } from 'react';
 import { action } from '@storybook/addon-actions';
```

Comment:
Would it be possible to use the actions that come from controls here? In argTypes you could do something like 
```
 onSubmit: { action: 'onSubmit' },
```